databases/week2/Working with tasks.txt

```diff
@@ -0,0 +1,44 @@
+
+	Write the following sql queries:
+	1- Add a task with the these attributes: title, description, created, updated, dueDate, statusID, userID
+		INSERT INTO `task` (`title`, `description`,`created`, `updated`, `due_Date`,`status_ID`, `user_ID`) 
+		VALUES ('complete_homework_week 2', 'Its about db', '2019-11-18 19:31:00','2019-11-18 20:31:00','2019-12-18 20:31:00', '3','8');
+
+	2- Change the title of a task with these attributes: taskID, newTitle
+		update task set title = 'Do some shopping for dark winter' where id = 1;
+
+	3-Change the task due date with these attributes: taskID, newDueDate
+		update task SET due_date = '2019-12-31 20:58:03' WHERE task.id = 1;
+
+	4-Change the task status with these attributes: taskID, newStatus
+		UPDATE task SET status_id = 2 WHERE id = 1;
+
+	5-Mark a task as complete with this attribute: taskID
+		UPDATE task SET status_id = 3 WHERE id = 5;
+
+	6- Delete task with this attribute: taskID
+		DELETE FROM task WHERE id = 36;
+
+
+	SCHOOL DATABASE!!

```

Comment:
Hi @Nazaqat 

The first part of your homework is great, but i would suggest that you save the SQL queries as a sql file instead of txt.  That way github shows the syntax with colors and it's easier to see what is going on. You can still put text inside the file as comments like this:

```sql
-- This is a comment
SELECT * FROM task;
```

Also i think you forgot to add the create table statement for the student table.